We have problem using hardcoded directories like /usr/local here
which will be checked for cross builds. This is a special case which
is valid for AIX only. We do not have AIX as one of our supported
build host or target. Therefore we get rid of the hardcoded paths
and make life easier for cross compilation process.

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Inappropriate [Upstream does care for AIX while we may not]

Index: coreutils-8.14/m4/getloadavg.m4
===================================================================
--- coreutils-8.14.orig/m4/getloadavg.m4	2011-09-19 08:09:24.000000000 -0700
+++ coreutils-8.14/m4/getloadavg.m4	2011-10-19 21:42:00.385533357 -0700
@@ -41,16 +41,6 @@
        [LIBS="-lutil $LIBS" gl_func_getloadavg_done=yes])
    fi
 
-   if test $gl_func_getloadavg_done = no; then
-     # There is a commonly available library for RS/6000 AIX.
-     # Since it is not a standard part of AIX, it might be installed locally.
-     gl_getloadavg_LIBS=$LIBS
-     LIBS="-L/usr/local/lib $LIBS"
-     AC_CHECK_LIB([getloadavg], [getloadavg],
-                  [LIBS="-lgetloadavg $LIBS" gl_func_getloadavg_done=yes],
-                  [LIBS=$gl_getloadavg_LIBS])
-   fi
-
    # Set up the replacement function if necessary.
    if test $gl_func_getloadavg_done = no; then
      HAVE_GETLOADAVG=0
